examples/notebooks/Watch_Warning_and_Advisory_Plotting.ipynb

```diff
@@ -0,0 +1,721 @@
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DataAccessLayer.changeEDEXHost(\"edex-cloud.unidata.ucar.edu\")\n",
+    "request = DataAccessLayer.newDataRequest()\n",
+    "request.setDatatype(\"warning\")\n",
+    "params = [\"phensig\", \"sig\"]\n",
+    "request.setParameters(*(params))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Significance (Sig) Constants\n",
+    "\n",
+    "The two parameters we're requesting for our warning objects are ***phensig*** and ***sig*** where phensig is styled \"XX.Y\" and sig is \"Y\".  The constants in this section correlate the ***sig*** to what type of message it is (what significance it is)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "WATCH_SIG = 'A'\n",
+    "WARN_SIG = 'W'\n",
+    "ADVIS_SIG = 'Y'\n",
+    "STATEM_SIG = 'S'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filter by Time\n",
+    "\n",
+    "Here we decide how much data we want to pull from EDEX.  By default we'll request 12 hours, but that value can easily be modified by [adjusting the `timedelta(hours = 12)`](https://docs.python.org/3/library/datetime.html#timedelta-objects) in line `2`.  The more data we request, the longer the next section will take to run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get records from the last 12 hours\n",
+    "lastHourDateTime = datetime.utcnow() - timedelta(hours = 12)\n",
+    "start = lastHourDateTime.strftime('%Y-%m-%d %H:%M:%S')\n",
+    "end = datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S')\n",
+    "\n",
+    "beginRange = datetime.strptime( start , \"%Y-%m-%d %H:%M:%S\")\n",
+    "endRange = datetime.strptime( end , \"%Y-%m-%d %H:%M:%S\")\n",
+    "timerange = TimeRange(beginRange, endRange)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use the Data!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get the Data\n",
+    "\n",
+    "Now that we have our `request` and TimeRange `timerange` objects ready, it's time to request the data array from EDEX.\n",
+    "\n",
+    "<br>\n",
+    "<div class=\"alert-info\">\n",
+    "<b>Note:</b> Above we set <b>timerange</b> to be 6 hours worth of data.  This can return on the order of ~2000 records and can take a little while to run.\n",
```

Comment:
Should this be 12 hours worth of data instead of 6?

Reply:
Yes!  This is supposed to say 12... and ~2000 is the number of records for 12 hours.  Updating that now.